docs/build/apps/wallet/sep24.mdx

```diff
@@ -347,18 +347,61 @@ While `Watcher` class offers powerful tracking capabilities, sometimes it's requ
 <CodeExample>
 
 ```kt
-val transaction = sep24.getTransactionBy(token, id = "transaction id")
+// "id" is the actual Anchor transaction id, all transactions should have it.
+val transaction = sep24.getTransactionBy(authToken, id = "transaction id")
+
+// "stellarTransactionId" (aka "stellar_transaction_id" on the SEP spec)
+// is the hash of the Stellar network transaction payment related to this
```

Comment:
Would it make sense to expand on "is the hash of the Stellar network transaction payment"?
Do we think the reader assumes sha256 already or should we add a snippet?

Reply:
@aristidesstaffieri good point, I've added a change here: https://github.com/stellar/stellar-docs/pull/930/commits/fd92a204d27510ff467f29580c5aed8270c22429
let me know what you think - thanks!

Reply:
lgtm 👍